dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -736,6 +736,8 @@ public void sortTopLevelCategories()  throws DotDataException {
 	}
 
 	public void sortChildren(String inode)  throws DotDataException {
+        inode = SQLUtil.sanitizeParameter( inode );

```

Comment:
Oscar, this is ok, but a better fix would be to make this method use a PreparedStatement instead of just injecting the inode value

https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategorySQL.java#L39